operations/boot_orchestration/Determine_Which_BOS_Session_Booted_A_Node.md

```diff
@@ -0,0 +1,110 @@
+# Determine Which BOS Session Booted a Node
+
+This guide is split into BOS Version 1 (V1) and BOS Version 2 (V2) sections
+because the procedures are different for the two different versions.
+
+## BOS Version 1 (V1)
+
+To determine which BOS Session booted or rebooted a node, query the node's
+kernel boot parameters. They contain a string 'bos_session_id' which identifies
+which BOS session booted or rebooted the node. Then, use this BOS Session ID
+to describe the BOS Session, which identifies the BOS Session template used.
+
+### Query the Node
+
+(`ncn-mw#`) Set an environment variable to the xname of the node in question. For example:
+
+```
+export NODE_XNAME="x3000c0s19b4n0"
+```
+
+(`ncn-mw#`) Query the node in question via ssh. The returned value will be the session ID.
+
+```bash
+BOS_SESSION_ID=$(ssh $NODE_XNAME "cat /proc/cmdline" | awk -v RS=" " -F "=" '{if ($1 == "bos_session_id") { print $2; }}')
+echo $BOS_SESSION_ID
+```
+
+The output will be the BOS session ID. For example:
+
+```
+4b6744ee-837f-4f60-9051-897aed6c7623
+```
+
+### Query BOS
+
+(`ncn-mw#`) Ask BOS to describe this session.
+
+'''bash
+cray bos v1 session describe ${BOS_SESSION_ID} --format json
+'''
```

Comment:
````suggestion
```bash
cray bos v1 session describe ${BOS_SESSION_ID} --format json
```
````

Not sure if this was my mistake in my suggestions or what, but single quotes instead of backticks were causing most of the spellchecker failures here.